base/system/diskpart/diskpart_msg.mc

```diff
@@ -7219,84 +7237,87 @@ Example:
     SET ID=ebd0a0a2-b9e5-4433-87c0-68b6b72699c7
 .
 Language=Romanian
-    Changes the partition type field for the partition with focus.
+    Schimbă registru tipului partiției pentru partiția în cauză.
```

Comment:
Please check the grammar. There is a disagreement on definitiveness between "registru" and "tipul partiției".